forge-gui/res/cardsfolder/upcoming/the_five_doctors.txt

```diff
@@ -0,0 +1,8 @@
+Name:The Five Doctors
+ManaCost:5 G
+Types:Sorcery
+K:Kicker:5
+A:SP$ ChangeZone | Origin$ Library | OriginAlternative$ Graveyard | Destination$ Hand | DestinationAlternative$ Battlefield | DestAltSVar$ MANDATORY Count$TimesKicked | ChangeType$ Doctor | ChangeNum$ 5 | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ0 | SubAbility$ DBChangeZone | SpellDescription$ Search your library and/or graveyard for up to five Doctor cards, reveal them, and put them into your hand. If you search your library this way, shuffle. If this spell was kicked, put those cards onto the battlefield instead of putting them into your hand.
```

Comment:
One of the Origins should be both zones
so you can combine cards from them

Reply:
Hmm, in testing it already allowed me to mix and match cards from both zones as long as I indicated that I wanted to search both... Forge did a prompt for each zone, which seems correct with the "and/or" ? I'm comparing this to something like Ecological Appraisal.